decidim-core/spec/types/area_api_type_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "decidim/api/test/type_context"
+
+module Decidim
+  module Core
+    describe AreaApiType do
+      include_context "with a graphql type"
+
+      let(:model) { create(:area) }
```

Comment:
if removed from factories, create the type here:
```suggestion
      let(:model) { create(:area, :area_type: area_type, organization: area_type.organization) }
      let(:area_type) { create(:area_type):
```